homeassistant/components/opengarage/cover.py

```diff
@@ -58,29 +56,29 @@
 
 async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
     """Set up the OpenGarage covers."""
-    covers = []
     devices = config.get(CONF_COVERS)
-
     for device_config in devices.values():
-        opengarage_url = (
+        device_config[CONF_HOST] = (
             f"{'https' if device_config[CONF_SSL] else 'http'}://"
-            f"{device_config.get(CONF_HOST)}:{device_config.get(CONF_PORT)}"
+            f"{device_config.get(CONF_HOST)}"
         )
```

Comment:
It looks like port is used in `validate_input` in the config flow

Do we need to keep it here?

Reply:
It is included in device_config.
I only parse the config to the new format. Moved the part to `async_step_import` to make it more obvious.